Representable/representable
# Login redesign

**changes**
- Authentication page now features the login form and signup form side by side with a new appearance
- Authentication redirection now features an alert at the top telling the user that login or signup is required.
- Authentication errors results in a red banner at the top of the form where the error occurs
- Username is now removed from the authentication process entirely: https://github.com/Representable/representable/issues/441#issue-722563335

**to test**
- Click login or signup button. If you click the login button, the focus should be on the email textbox of the login form. If you click the signup form should be on the email textbox of the signup form.
- On small tablet and mobile view, if you click on the "signup" and "login" links at the bottom of each form, the screen should be scrolled to and the focus should be on that respective form.
- To test the redirection: Make sure that you are logged out then click on a feature that requires authentication. If you put in the wrong information in either form then you should be redirected to the same page with the error displayed. When you put in the correct credentials you should be redirected to whatever page you were trying to access.

**screenshots**

**Note: The banner at the top of the redirection page should now read *"Let's have you sign up or log in before you start..."* not as pictured. Also, the signup form and login form positions are flipped from what is shown in pictures. However, the designs are the same.**

<img width="242" alt="login-redesign-redirect-mobile" src="https://user-images.githubusercontent.com/28410610/100711369-09db9b80-3366-11eb-813d-2ead9e9db76a.png">
<img width="324" alt="login-redesign-redirect-tablet" src="https://user-images.githubusercontent.com/28410610/100711371-0a743200-3366-11eb-914f-e740f93ed9ce.png">
<img width="1542" alt="login-redesign-redirect-laptop" src="https://user-images.githubusercontent.com/28410610/100711374-0b0cc880-3366-11eb-82f2-1c24425f8410.png">
<img width="243" alt="login-redesign-mobile-p2" src="https://user-images.githubusercontent.com/28410610/100711378-0c3df580-3366-11eb-8d68-78e193d65161.png">
<img width="242" alt="login-redesign-mobile-p1" src="https://user-images.githubusercontent.com/28410610/100711379-0c3df580-3366-11eb-9cbe-a6127d401494.png">
<img width="324" alt="login-redesign-regtablet" src="https://user-images.githubusercontent.com/28410610/100711381-0cd68c00-3366-11eb-9f33-89547ad52779.png">
<img width="1542" alt="login-redesign-laptop" src="https://user-images.githubusercontent.com/28410610/100711385-0cd68c00-3366-11eb-991a-b681d3305412.png">
